cmd/http/deadlineconn.go

```diff
@@ -24,10 +24,8 @@ import (
 // DeadlineConn - is a generic stream-oriented network connection supporting buffered reader and read/write timeout.
 type DeadlineConn struct {
 	net.Conn
-	readTimeout            time.Duration // sets the read timeout in the connection.
-	writeTimeout           time.Duration // sets the write timeout in the connection.
-	updateBytesReadFunc    func(int)     // function to be called to update bytes read.
-	updateBytesWrittenFunc func(int)     // function to be called to update bytes written.
+	readTimeout  time.Duration // sets the read timeout in the connection.
+	writeTimeout time.Duration // sets the write timeout in the connection.
```

Comment:
Now it looks simpler by getting rid of updateBytesReadFunc and updateBytesWrittenFunc

Reply:
We should have gotten rid of this some time ago.